src/routes/membrosRoutes.ts

```diff
@@ -1,10 +1,19 @@
 import { Router } from 'express';
 import { MembrosController } from '../controller/membrosController';
 // import { authenticateJWT, hasPermission } from '../middlewares/authMiddleware';
+import { validarEmail } from '../middlewares/validarEmail';
 
 const membrosRouter = Router();
 const membrosController = new MembrosController();
 
+membrosRouter.post(
+  '/',
+  validarEmail,
+  //  authenticateJWT,
```

Comment:
Se não for utilizar no futuro, pode remover essas linhas, mas caso for utilizar é uma boa prática deixar um comentário FIXME ou TODO numa linha acima indicando o porque desse código ser comentado

Reply:
Ainda será usado